test/verify/check-storage-stratis

```diff
@@ -41,6 +41,11 @@ def get_stratis_stop_type_opt(execute):
     return ""
 
 
+def create_pool_key(machine, keyname, passphrase):
+    # this is a bit complicated, see https://bugzilla.redhat.com/show_bug.cgi?id=2246923
+    machine.execute(f"echo -n '{passphrase}\n{passphrase}' | stratis key set --capture-key {keyname}")
```

Comment:
`echo -n {passphrase} | stratis key set --keyfile-path /dev/stdin {keyname}` should work. I think you didn't use `-n`, and your passphrases would have an additional newline.

Reply:
Ah, I really meant `echo -e` here, but as I simultaneously forgot to mark it as a raw string, it worked by accident :see_no_evil: 

I tried with `echo -n | --keyfile-path`, but that breaks TestStorageStratisNBDE.testBasic for me locally. But giving it a shot here.

Reply:
Nevermind, that failure is https://github.com/cockpit-project/bots/issues/4796 -- why is everything so broken here :cry: 

So this does work on [f38](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19549-20231031-151509-68d2b46a-fedora-38-storage/log.html), [f39](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19549-20231031-151519-68d2b46a-fedora-39-storage/log.html) and [f39/updates-testing](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19549-20231031-151510-68d2b46a-fedora-39-updates-testing-storage/log.html). I updated the commit again.